src/app/renderer/components/form/Select/Select.tsx

```diff
@@ -1,15 +1,14 @@
 import * as React from 'react';
 import * as ReactDOM from 'react-dom';
-import ReactSelect, {OnChangeHandler, Option, Options} from 'react-select';
+import ReactSelect from 'react-select';
 
 require('./SelectStyles.scss');
```

Comment:
everything inside SelectStyles.scss is not needed with the new react-select version anymore. So you can probably just remove it. One thing I noticed though is that the dropdown is now hidden in some cases (See attached image). I guess the same applies to the Autocomplete component.
<img width="393" alt="hidden" src="https://user-images.githubusercontent.com/864455/50099284-094fa600-021e-11e9-95d8-7e9c96027cdf.png">

Reply:
I think I fixed all the problems except this one, I don't understand why that is happening so I could not fix it :/